### PR TITLE
Gminer support

### PR DESCRIPTION
**Pull request by memedmusic**
Adds support for gminer because some person in the discord server asked for it. 

Since gminer is sussy i had to rearrange a few things

Gminer only supports stratum, so I had to make it change the ethproxy pools to stratum. Gminer also requires a separate port argument, so I had to do more sussy code to fix that

Tested to work on windows
this took way too long to work out

### CODE REVIEW

**Comment by memedmusic** [comment]
Also btw to get gminer to work users will need to update saladbind

File: internal/miners.json [hunk 1]
```diff
@@ -127,6 +127,34 @@
                 "pool": "-o",
                 "pass": "-p"
             }
+        },  "GMiner": {
```
Comment (Vukkyy) [inline internal/miners.json:130]:
```suggestion
        },  
        "GMiner": {
```
absolutely horribly written would not ercommend

this is how you do it right

**Review by Vukkyy — CHANGES_REQUESTED** [review]

File: internal/miners.json [hunk 2]
```diff
@@ -127,6 +127,34 @@
...
+            "miner": "GMiner",
+            "download": {
+                "win32": "https://github.com/develsoftware/GMinerRelease/releases/download/2.74/gminer_2_74_windows64.zip",
```
Comment (actuallyasmartname) [inline internal/miners.json:133]:
```suggestion
                "win32": "https://github.com/develsoftware/GMinerRelease/releases/download/2.75/gminer_2_75_windows64.zip",
```
gminer updated

**Review by actuallyasmartname — CHANGES_REQUESTED** [review]

File: internal/miners.json [hunk 3]
```diff
@@ -127,6 +127,35 @@
                 "pool": "-o",
                 "pass": "-p"
             }
+        },  
+        "GMiner": {
+            "miner": "GMiner",
+            "download": {
+                "win32": "https://github.com/develsoftware/GMinerRelease/releases/download/2.75/gminer_2_75_windows64.zip",
+                "linux": "https://github.com/develsoftware/GMinerRelease/releases/download/2.74/gminer_2_74_linux64.tar.xz"
```
Comment (actuallyasmartname) [inline internal/miners.json:135]: ops i forgot to comment on the linux version

**Review by actuallyasmartname — CHANGES_REQUESTED** [review]

**Comment by actuallyasmartname** [comment]
also update the linux version, idk how i missed that

**Pull request closed without merging**